### PR TITLE
P1 - Grid Component

### PR DESCRIPTION
**Pull request by bharati-21**
- Added grid component with following variations:
  - Grid with 2 columns
  - Grid with 3 columns
  - Grid with 4 columns

### CODE REVIEW

**Review by NeerajVodala — APPROVED** [review]
Looks good to me.
All the best.

**Comment by bharati-21** [comment]
> Looks good to me. All the best.

Thank you! Wishing you the same. 👍🏽

File: docs/card/index.html [hunk 1]
```diff
@@ -72,6 +72,9 @@ <h5 class="sidebar__head">
                         <li class="link--active">
                             <a href="/docs/card/" class="link text-link">Card</a>
```
Comment (raghav6202) [inline docs/card/index.html:73]: Two hyphens here. Might be a typo

**Review by raghav6202 — APPROVED** [review]
Well written code. You have used two hyphens in naming certain classes  didn't quite get that. Nice U.I looks good to merge👍.

**Comment by bharati-21** [comment]
> Well written code. You have used two hyphens in naming certain classes didn't quite get that. Nice U.I looks good to merge👍.

Thank you!
Yes, I used the BEM convention for the documentation website.

**Pull request merged**